server/src/cli.ts

```diff
@@ -27,6 +27,11 @@ if (!(program.stdio || program.socket || program.nodeIpc)) {
     process.exit(1);
 }
 
+if (program.tsserverLogFile && !program.tsserverLogVerbosity) {
+  console.error('--tsserver-log-file cannot be used without --tsserver-log-verbosity');
```

Comment:
it should default to 'normal' instead of failing.

Reply:
Updated